modules/ipi-manual-migration-to-customdeploy.adoc

```diff
@@ -0,0 +1,45 @@
+// This is included in the following assemblies:
+//
+// ipi-install-post-installation-configuration.adoc
+
+:_content-type: PROCEDURE
+[id="manual-migration-to-new-customdeploy-install-method_{context}"]
+
+= Manual migration to new customDeploy install method
+
+A new deployment method introduced in 4.10 allows you to customize the network configuration (`networkConfig`) in the `install-config.yaml` file by host during the install and provisioning process. You can also set static IPs per host and additional advanced network configurations.
+
+When you upgrade to {product-title} 4.10, the new deployment method is not automatically upgraded and therefore you need to perform the following manual steps.
+
+
+.Procedure
+
+. Log in to `oc` as a user with `cluster-admin` permission.
+
+. Find out what machineSets exists:
++
+[source,terminal]
+----
+$ oc get machinesets -A
+----
+
+. Edit each machineSet:
++
+[source,terminal]
+----
+$ oc edit machineset <machineset> -n openshift-machine-api
+----
+
+.. Amend to include the following:
```

Comment:
Wondering if the instructions should include a comment to the user on what the change is. e.g.
"note the key change is to remove the image checksum/url and add the customDeploy field"